packages/slice-machine/components/Button/index.tsx

```diff
@@ -47,7 +47,7 @@ export const Button: React.FunctionComponent<ButtonProps> = ({
     }}
     type={type}
     form={form}
-    disabled={disabled}
+    disabled={disabled || isLoading}
```

Comment:
Imo this makes sense since the button design is the same in both states (except for the cursor which is already overwritten above).

This then allows us to use the `&:disabled` styling from ThemeUI for each variant

Reply:
hey, makes sense to me as well, just the design was specifying a scenario where the button was loading and not disabled.

Reply:
But i think here since loading and disabled are independent, we loose one possible case of a button being activated multiple times.
Atm it doesn't change anything tbh because we don't use that case but, just saying :)